app/javascript/react/components/app.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export const App = (props) => {
+  return (<h1>Make It So React</h1>)
+}
```

Comment:
Based on this component, it doesn't look like any of your React code is being rendered to the page?

Reply:
Ohhh It looks like you created a container `App`. Please remove this extraneous `app` file in `Components`!